corehq/apps/data_interfaces/management/commands/fix_bad_reassigned_owner_ids.py

```diff
@@ -62,18 +65,26 @@ def handle(self, domains, **options):
 
 
 def get_valid_owner_ids(domain):
-    return set(itertools.chain(
-        get_case_sharing_group_ids(domain),
-        get_location_ids(domain),
-        get_user_ids(domain)
-    ))
+    while True:
```

Comment:
Is there a scenario where the ESError is permanent? If so, we would want a max retries.

Reply:
Seems worth checking for - will add in follow up PR